examples/language/lib/class_modifiers/ex5/sealed.dart

```diff
@@ -7,16 +7,21 @@ class Truck implements Vehicle {}
 
 class Bicycle extends Vehicle {}
 
-// ERROR: Can't be instantiated.
+// ERROR: `Vehicle` can't be instantiated because
+// it is marked `sealed` and therefore, implicitly abstract.
 Vehicle myVehicle = Vehicle();
 
-// Subclasses can be instantiated.
+// Subclasses of a sealed class can be instantiated unless also restricted.
 Vehicle myCar = Car();
 
-String getVehicleSound(Vehicle vehicle) {
-  // ERROR: The switch is missing the Bicycle subtype or a default case.
-  return switch (vehicle) {
-    Car() => 'vroom',
-    Truck() => 'VROOOOMM',
-  };
+extension VehicleSounds on Vehicle {
+  String get sound {
+    // ERROR: The switch does not exhaustively account for
+    // all possible instances of a `Vehicle`.
+    // In this case, a `Vehicle` with a run-time type of `Bicycle`.
```

Comment:
"does not account for" might be too broad to unambiguously imply the non-match, but we could make it explicit:

```suggestion
    // In this case, a `Vehicle` with a run-time type of `Bicycle` will not
    // match any of the cases.
```